components/02-molecules/pop-out-image/_pop-out-image.scss

```diff
@@ -1,5 +1,16 @@
 @use '../../00-tokens/tokens';
 
+.pop-out-image {
+  @include tokens.spacing-page-inner;
+
+  // Clear the float in case the next component is an image or something.
```

Comment:
:+1:

Reply:
I'm pretty sure I added those wrapping ifs to the JavaScript... Maybe in the main menu PR... ? So the console errors should go away soon

Reply:
Yep I see that now :fist_right: :fist_left: